pmm-app/tests/pages/dashboardPage.js

```diff
@@ -678,4 +681,13 @@ module.exports = {
     I.waitForElement(filterNameSelector, 30);
     I.click(filterNameSelector);
   },
+
+  async verifyTimeRange(timeRange) {
+    const timeRangeGrabbed = await I.grabTextFrom(this.fields.timeRangePickerButton);
+    assert.equal(
```

Comment:
I think what we can do is, change this to getTimeRange and move the assertion to the scenario because this is not a common assertion and will not be used by all, but getTimeRange makes much more sense to me.